README.md

```diff
@@ -0,0 +1,26 @@
+# flashpointvm
```

Comment:
README looks good, but I'd prefer if it were moved to a separate PR.

Reply:
Okay, sounds good.
Resolved in 15a4561.